Common subdirectories: OneWire/examples and OneWirePa/examples
diff -u OneWire/OneWire.cpp OneWirePa/OneWire.cpp
--- OneWire/OneWire.cpp	2011-11-01 19:52:08.000000000 +0100
+++ OneWirePa/OneWire.cpp	2013-09-21 21:27:26.776910882 +0200
@@ -99,17 +99,24 @@
 #include "OneWire.h"
 
 
+OneWire::OneWire() {
+
+}
+
 OneWire::OneWire(uint8_t pin)
 {
-	pinMode(pin, INPUT);
-	bitmask = PIN_TO_BITMASK(pin);
-	baseReg = PIN_TO_BASEREG(pin);
+    setPin(pin);
+}
+
+void OneWire::setPin(uint8_t pin) {
+    pinMode(pin, INPUT);
+    bitmask = PIN_TO_BITMASK(pin);
+    baseReg = PIN_TO_BASEREG(pin);
 #if ONEWIRE_SEARCH
-	reset_search();
+    reset_search();
 #endif
 }
 
-
 // Perform the onewire reset function.  We will wait up to 250uS for
 // the bus to come high, if it doesn't then it is broken or shorted
 // and we return a 0;
diff -u OneWire/OneWire.h OneWirePa/OneWire.h
--- OneWire/OneWire.h	2011-11-01 19:52:10.000000000 +0100
+++ OneWirePa/OneWire.h	2013-09-21 21:27:26.776910882 +0200
@@ -93,8 +93,11 @@
 #endif
 
   public:
+    OneWire();
     OneWire( uint8_t pin);
 
+    void setPin(uint8_t pin);
+
     // Perform a 1-Wire reset cycle. Returns 1 if a device responds
     // with a presence pulse.  Returns 0 if there is no device or the
     // bus is shorted or otherwise held low for more than 250uS
